frontend/src/store/modules/auth/actions.js

```diff
@@ -0,0 +1,30 @@
+export default {
+    login: (context, user) => {
```

Comment:
use explicit param definition where it's possible like, for example, instead of `context` better to use `{commit}`, since it's only one param that is used from context